skimage/restoration/tests/test_denoise.py

```diff
@@ -224,7 +224,9 @@ def test_denoise_bilateral_multidimensional():
 
 def test_denoise_bilateral_nan():
     img = np.full((50, 50), np.NaN)
-    out = restoration.denoise_bilateral(img, multichannel=False)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        out = restoration.denoise_bilateral(img, multichannel=False)
```

Comment:
Here also, why not `expected_warnings`?

Reply:
I'm pretty sure for that one, it gives a warning on 3.X and not on 2.7. Probably because of a divide by 0 or Nan rule.